server/utils/mojPagination/pagination.ts

```diff
@@ -0,0 +1,87 @@
+import { ApiPageRequest, ApiPageResponse } from '../../data/ApiData'
+import PageResponse from './pageResponse'
+
+type PageLink = {
+  text: string
+  href: string
+}
+
+type NumberedPageLink = {
+  text: string
+  href: string
+  selected?: boolean // default = false
+}
+
+type ResultsMetaData = {
+  from: number
+  to: number
+  count: number
+}
+
+type PageViewModel = {
+  results: ResultsMetaData
+  previous: PageLink | null
+  next: PageLink | null
+  items: NumberedPageLink[] | null
+}
+
+export function pageRequestFrom(pageSize: number, pageNumber: number): ApiPageRequest {
+  return {
+    size: pageSize,
+    number: pageNumber - 1,
```

Comment:
So this is now where we change the index - and its to a concrete ApiPageRequest - Cool